README.md

```diff
@@ -10,9 +10,9 @@ A golang library for the [COSE specification][cose-spec]
 
 **Current Release**: [go-cose alpha 1][release-alpha-1] 
 
-While the project is *currently* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, the veraison and mozilla maintainers have agreed to decouple the fork, retire the mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
+The project was *initially* forked from the  upstream [mozilla-services/go-cose][mozilla-go-cose] project, however the Veraison and Mozilla maintainers have agreed to retire mozilla-services/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

Comment:
```suggestion
The project was *initially* forked from the upstream [mozilla-services/go-cose][mozilla-go-cose] project, however the Veraison and Mozilla maintainers have agreed to retire mozilla-services/go-cose and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

Reply:
I think the existing version carried with project term is well formed. I will retain it like this!

Reply:
ok, cool.  Then add a "the" after "retire"

Reply:
Done!